pybossa/forms/forms.py

```diff
@@ -267,6 +267,9 @@ def in_range(self):
         return self.min_seconds <= minutes*60 + seconds <= self.max_minutes*60
 
 
+class ProgressReminderForm(Form):
+    remaining = IntegerField(lazy_gettext('Notify when incomplete tasks drops to the following number'))
```

Comment:
Perhaps, consider rephrasing as follows?

`Notify when the number of remaining tasks is less than or equal to`

Reply:
Is this field required?

Reply:
this field is not required, can be empty